low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -257,7 +257,7 @@ def _create_vhost_if_necessary(virtual_host, product):
             response = _send_command(command)
 
         vhosts = response.split('\n')
-        assert vhosts[0] == 'Listing vhosts ...'
+        # assert vhosts[0] == 'Listing vhosts ...'
```

Comment:
We should convert these into log prints, i.e. print vhosts or other values if not done already. In case config fails we will have this data for inspection.

Reply:
Done.